### ISSUE TITLE
Styling issue row class

### ISSUE THREAD

**Issue by JustCarmen** [fisharebest/webtrees#1352]
With commit https://github.com/fisharebest/webtrees/commit/f21898e8fef3eae457cf18e6dae5a58c3103a069 a common style is set that effects all row classes throughout the site. 

`.wt-main-container .row {
	flex-wrap: wrap;
	margin: 0;
}`

The margin of 0 is now set on all form rows, which gives them a weird look.

According to the comment in the css file this style should only affect the homepage.

**Comment by fisharebest** [comment]
IIRC, that was written for an earlier version of the bootstrap grid.

#1337 exists to review all this.

**Comment by fisharebest** [comment]
Looking at these, I'm guessing that they were added against bootstrap4.6alpha, and are not needed with bootstrap4.beta.

For example, this appears to be redundant, as the class `nav` already contains the same rules.

```
.nav {
	display: flex;
    flex-wrap: wrap;
}
```

**Comment by fisharebest** [comment]
Bootstrap4-beta now provides lots of utility classes, such as `w-100` (width: 100%).

This allows you to easily apply formatting to specific items, without needing global rules.

**Comment by fisharebest** [comment]
@makitso 

Have you seen the bootstrap4 utility classes?  e.g.
https://getbootstrap.com/docs/4.0/utilities/flex/

They are designed for the sort of change you made.

I'm not 100% sure of the problem you are solving, but you could add `class="d-sm-flex flex-column"` on the elements themselves.

The idea is that using these helper classes for the "scaffolding" allows you to keep the main CSS file cleaner, and focused more on the "styling"

**Comment by makitso** [comment]
@fisharebest  One of the devices I test with is my mobile phone with a screen w of 412.  These changes were intended to improve displays at that width.  I guess it's safe to say that I am not that skilled when it comes to flex and all it's subtleties. I need to work on this :-)

**Comment by fisharebest** [comment]
@makitso - I don't really know what problem you are trying to fix, or what you are trying to achieve with this change. "Improve" covers a lot of things...

**Comment by makitso** [comment]
I have been testing the home page and My page with various combinations of blocks for smaller screens e.g. 400 vw.  For example, on the home page, all blocks were having their left and right margins cutoff due to a bootstrap styling of:
`row {
    display: flex;
    flex-wrap: wrap;
    margin-left: -15px;
    margin-right: -15px;
}
`
So, I fixed this with the following that was included in the code that you deleted.  I have added it back in and reduced the scope with a @media statement in the _common style.css.
`.wt-main-container .row {
    flex-wrap: wrap;
    margin: 0;
}`
